CHANGELOG.md

```diff
@@ -4,6 +4,26 @@
 
 ### Internal
 
+- [In Progress: 22 Sep 2021] Custom Drug Entry Sheet UI Improvements
```

Comment:
> Custom Drug Entry Sheet UI Improvements
this is already completed and already in the demo changelog. Can we remove this from here? There must have been some mistake while rebasing somewhere which is why this is here.